Gemfile

```diff
@@ -4,6 +4,8 @@ source "https://rubygems.org"
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby '>=2.6.10'
 
-gem "cocoapods", ">= 1.15.2"
+# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
+# bound in the template on Cocoapods with next React Native release.
+gem 'cocoapods', '>= 1.13', '< 1.15'
```

Comment:
What kind of bug are we talking about? 🤔 
I've used `cocoapods` `1.15.2` and I was able to correctly build the app both on simulator and on physical device.

```zsh
▸ Build Succeeded
success Successfully built the app

➜ git diff

diff --git a/ios/Podfile.lock b/ios/Podfile.lock
index b97e73ae4..df90bd598 100644
--- a/ios/Podfile.lock
+++ b/ios/Podfile.lock
@@ -1164,4 +1164,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d291c9001dab31affbfc57fa39ac51377e3ff0d3
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Reply:
Forget it! I didn't revert the whole. Podfile.lock.

Reply:
react-native suggests to stick on cocoapods 1.14.x since there's been some issues on 1.15... that's why I opted to downgrade it